pkgs/development/tools/build-managers/goredo/default.nix

```diff
@@ -50,6 +53,7 @@ buildGoModule rec {
   outputs = [ "out" "info" ];
 
   meta = with lib; {
+    outputsToInstall = [ "out" "info" ];
```

Comment:
Do we really need info files all the time?

Reply:
The info file is the only form of documentation. Should I leave it out?

Reply:
outputsToInstall should only install the info output if the program is not working without it. If users opt out of installing info files then they should not be enforced here

Reply:
Right, I removed it.